flow360/component/flow360_params/flow360_output.py

```diff
@@ -423,8 +426,8 @@ def validate_monitor(cls, values):
 class MonitorOutput(Flow360BaseModel):
     """:class:`MonitorOutput` class"""
 
-    monitors: Optional[Monitors] = pd.Field()
-    output_fields: Optional[CommonOutputFields] = pd.Field(alias="outputFields")
+    monitors: Optional[Monitors] = pd.Field(default={})
```

Comment:
Why default to empty dict? This is optional parameter, so I think monitors=None make more sense, so no default at all.

Reply:
monitors is actually not optional I will remove that.

Instead I will make Monitor/Slice/...Output optional with no defaults since they are not required.